src/compile/output/mod.rs

```diff
@@ -833,6 +914,11 @@ impl<'a> fmt::Display for OutputDiagnostics<'a> {
                         // from a test file we skip
                         ignored =
                             self.is_test(&source_location.file) && (code == 1878 || code == 5574);
+
+                        // we ignore warnings coming from ignored files
+                        ignored |= self
+                            .ignored_file_paths
+                            .starts_with(&[PathBuf::from(&source_location.file)]);
```

Comment:
Path::new, also I don't think this is right? Why `starts_with` on a list of ignored paths?

Reply:
it should be the other way around yes